src/cascadia/TerminalControl/SearchBoxControl.idl

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+namespace Microsoft.Terminal.TerminalControl
+{
+    [default_interface] runtimeclass SearchBoxControl : Windows.UI.Xaml.Controls.UserControl
+    {
+        SearchBoxControl();
+        Boolean GoForward { get; };

```

Comment:
nit: since this isn't being used outside the `SearchBoxControl` itself, it doesn't really need to be part of the interface defined here, and the implementation can be `private`

Reply:
I wonder if you mean "GoForward" ? It is used in TermControl actually.